src/Integration.Vsix/SonarLintTagger/IIssueTracker.cs

```diff
@@ -24,8 +24,6 @@ namespace SonarLint.VisualStudio.Integration.Vsix
 {
     internal interface IIssueTracker
     {
-        void AddTagger(IssueTagger tagger);
-        void RemoveTagger(IssueTagger tagger);
```

Comment:
I've moved the responsibility for keeping track of the taggers-per-tracker to inside the tracker
i.e. the tracker will call add/remove itself, so it no longer requires publicly visible add/remove methods.